assets/js/modules/analytics-4/datastore/accounts.js

```diff
@@ -183,6 +183,8 @@ const baseActions = {
 
 			registry.dispatch( MODULES_ANALYTICS_4 ).setSettings( {
 				accountID,
+				propertyID: '',
+				webDataStreamID: '',
```

Comment:
This was necessary to avoid a situation where the webdatastream could be left in the "create" state and continue showing the input field while selecting a new account.